support/ebpf/Makefile

```diff
@@ -16,6 +15,9 @@ else
 $(error Unsupported architecture: $(NATIVE_ARCH))
 endif
 
+# Use a placeholder like '.' or '/' as the new prefix.
+REPRODUCIBLE_PREFIX := .
+
 # Valid values: amd64, arm64.
 TARGET_ARCH ?= $(NATIVE_ARCH)
 
```

Comment:
Below here is `BUILD_TYPE` type set, which is no longer relevant.
Also the `TRACER_NAME` should no longer depend on `BUILD_TYPE`
Could the makefile be further simplified from these now obsolete things?
Perhaps the Tracer name could be also made to not include the `release` in the filename? Just call it `tracer.ebpf.$(TARGET_ARCH)`?